### ISSUE TITLE
Implement integration tests that confirm specific Xcode versions work

### ISSUE THREAD

**Issue by cgrindel** [buildbuddy-io/rules_xcodeproj#247]
The project aims to define a matrix of Bazel versions and Xcode versions that work with `rules_xcodeproj`. The integration tests should exercise a basic set of functionality for each supported combination. As a general rule, the project will support the latest Xcode version, the previous release and the last major release. As of the writing of this issue, those versions are:
- Latest: 13.3
- Previous: 13.2.1
- Previous Major: 13.0

NOTE: The supported Bazel versions are already defined in `//:bazel_versions.yml`. 

- [ ] Create a file/macro that defines the supported Xcode versions. 
- [ ] Update the `ios_app` integration test to execute the supported combinations.

**Comment by brentleyjones** [comment]
We aren't using https://github.com/bazel-contrib/rules_bazel_integration_test anymore, but we should still do this, just at the BuildBuddy Workflows level.

**Comment by brentleyjones** [comment]
I don't think this should block the 1.0 release, so I've changed the milestone. Please tell me if you believe differently.
